publish/dir.props

```diff
@@ -17,28 +17,23 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PublishRid Include="win7-x64"/>
-    <PublishRid Include="win7-x86"/>
-    <PublishRid Include="win8-arm"/>
-    <PublishRid Include="win10-arm64"/>
-    <PublishRid Include="osx.10.12-x64"/>
     <PublishRid Include="ubuntu.14.04-x64"/>
     <PublishRid Include="ubuntu.16.04-x64"/>
-    <PublishRid Include="ubuntu.14.04-arm"/>
-    <PublishRid Include="ubuntu.16.04-arm"/>
     <PublishRid Include="ubuntu.16.10-x64"/>
-    <PublishRid Include="rhel.7-x64"/>
     <PublishRid Include="debian.8-x64"/>
-    <PublishRid Include="fedora.23-x64"/>
-    <PublishRid Include="fedora.24-x64"/>
-    <PublishRid Include="opensuse.42.1-x64"/>
-    <PublishRid Include="linux-x64"/>
-    <PublishRid Include="linux-arm"/>
-    <PublishRid Include="linux-arm64"/>
-    <PublishRid Include="win-x86"/>
-    <PublishRid Include="win-x64"/>
+    <PublishRid Include="linux-x64_portable"/>
+    <PublishRid Include="win-x86_portable"/>
```

Comment:
This publishrid itemgroup only gets used for checking the badge files. It's not actually a rid list like the others we use. https://github.com/dotnet/core-setup/blob/master/publish/dir.targets#L61 ID line to merge this Metadata in with the other rid lists, but that can happen later. If we eliminate the portable moniker, we can get rid of "_portable"